indy_node/test/upgrade/helper.py

```diff
@@ -133,7 +133,9 @@ def check_node_set_acknowledges_upgrade(
         node_ids=node_ids)
 
     for node in node_set:
-        node.upgrader.scheduledUpgrade = (version, 0, randomString(10))
+        node.upgrader.scheduledUpgrade = (version,
+                                          datetime.utcnow().replace(tzinfo=dateutil.tz.tzutc()),
```

Comment:
We can use `get_utc_epoch()` utility method for the same purpose

Reply:
`get_utc_epoch()` returns UTC timestamp (in seconds as `int` value) rather than `datetime`.